### PR TITLE
Implement fetch() function to request content from the server.

### PR DESCRIPTION
**Pull request by sofifch**
Learning about fetch()!

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -76,6 +76,8 @@ <h2 id="gettoknowme">Get To Know Me!</h2>
             <li><a href="#https://www.linkedin.com/in/sofia-fernandez-chacin-242787175/">LinkedIn</a>  <a 
             href="#https://github.com/sofifch">GitHub</a>  <a
```
Comment (rjkroege) [inline portfolio/src/main/webapp/index.html:77]: The `#` looks like a mistake to me. Is that valid syntax?

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -35,3 +35,9 @@ function myFunction() {
   var scrolled = (winScroll / height) * 100;
   document.getElementById("myBar").style.width = scrolled + "%";
 }
+
+async function getMessage() {
```
Comment (rjkroege) [inline portfolio/src/main/webapp/script.js:39]: Cool. Very modern JavaScript.

**Review by rjkroege — APPROVED** [review]

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (sofifch) [reply portfolio/src/main/webapp/index.html:77]: You are right, it is a mistake! Will fix it.

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (sofifch) [reply portfolio/src/main/webapp/script.js:39]: :)

**Pull request merged**